qcodes/dataset/experiment_settings.py

```diff
@@ -0,0 +1,60 @@
+"""Settings that are indirectly related to experiments."""
+
+from typing import Optional
+from qcodes.dataset.sqlite.connection import ConnectionPlus, atomic_transaction
+
+# The default experiment's exp_id. This changes to the exp_id of a created/
+# loaded experiment.
+default_experiment: Optional[int] = None
```

Comment:
Perhaps we should make the attribute private. @astafan8 what do you think?

Reply:
It is private by its nature, so we can do that.

Reply:
agree with making this variable private (adding underscore at the beginning of it's name)